hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -138,9 +141,11 @@ public boolean dropTable(TableIdentifier identifier, boolean purge) {
         dropTableData(ops.io(), lastMetadata);
       }
 
+      LOG.info("Table is dropped [{}]", identifier);
       return true;
 
     } catch (NoSuchTableException | NoSuchObjectException e) {
+      LOG.debug("Unable to drop missing table: [{}]", identifier, e);
```

Comment:
How about "Skipping drop, table does not exist: {}"?

Reply:
Done